Run daily job to send emails to suppliers about new questions and answers

idavidmcdonald opened the pull request:
This script will run everyday at 8am - https://github.com/alphagov/digitalmarketplace-scripts/pull/132

Trello: https://trello.com/c/2noI0EFB/472-3-tell-interested-suppliers-that-a-question-has-been-answered

TheDoubleK commented on job_definitions/notify_suppliers_of_new_questions_answers.yml:
```diff
@@ -0,0 +1,56 @@
+          wipe-workspace: false
+{% if environment == 'production' %}
+    triggers:
+      - timed: "0 8 * * *"
```
This is *exactly* the same time as the "notify_suppliers_of_dos_opportunities" job.

I'd be tempted to change both jobs to `H 8 * * *` (or `H 8 * * 1-5` for the opportunities one) so they're not both kicking off together.

TheDoubleK commented on job_definitions/notify_suppliers_of_new_questions_answers.yml:
```diff
@@ -0,0 +1,56 @@
+          . ./venv/bin/activate
+          pip install -r requirements.txt
+
+          if [ -n "SUPPLIER_IDS" ]; then
```
Should `SUPPLIER_IDS` have a `$` in front of it here? (I don't know for sure, but suspect yes).

idavidmcdonald replied:
Got spot

TheDoubleK commented on job_definitions/notify_suppliers_of_new_questions_answers.yml:
```diff
@@ -0,0 +1,56 @@
+            FLAGS="$FLAGS --dry-run"
+          fi
+
+          ./scripts/notify-suppliers-of-new-questions-answers.py '{{ environment }}' --api-token="$DM_DATA_API_TOKEN_{{ environment|upper }}" --mandrill-api-key="$MANDRILL_API_KEY_{{ environment|upper }}" $FLAGS
```
Is it too late to change the name of the script to include the word "and"? :trollface:

idavidmcdonald commented:
Thanks @TheDoubleK, I think we've addressed your comments. Personally I like keeping `{% set environments = ['production'] %}` even though there is only one environment, I think it's good for readability although you might argue that it's unneeded. Let me know if you feel strongly?

TheDoubleK commented:
I like leaving it in there too.  Nice one :+1:
